circle/functions

```diff
@@ -457,43 +460,54 @@ add_repo_to_helm() {
   helm repo add $REPO_NAME $REPO_URL
 }
 
-chart_update_image() {
-  local CHART_NEW_IMAGE_VERSION=${2#*:}
-  local CHART_CURRENT_IMAGE_VERSION=$(grep "^[ ]*image:[ ]*${2%:*}" ${1}/values.yaml)
-  local CHART_CURRENT_IMAGE_VERSION=${CHART_CURRENT_IMAGE_VERSION##*:}
+chart_update_image_tag() {
+  local CHART_PATH=${1}
+  local CHART_NEW_IMAGE_TAG=${2}
+  local CHART_IMAGE_REPOSITORY=${3}
+  # Look for repository matching our image. We want the tag property that must be below. Using xargs to trim the output.
+  # image:
+  #   repository: bitnami/wordpress
+  #   tag: 4.5.2
+  # 
+  local CHART_CURRENT_IMAGE_TAG=$(grep -A 1 "^[ ]*repository:[ ]*${CHART_IMAGE_REPOSITORY}" ${CHART_PATH}/values.yaml | tail -n 1 | xargs)
+  if [[ ${CHART_CURRENT_IMAGE_TAG} != *"tag"* ]]; then
+    error "Unable to determine current image tag"
+    exit 1
+  fi  
+  local CHART_CURRENT_IMAGE_TAG=${CHART_CURRENT_IMAGE_TAG##*:}
 
-  case $(vercmp $CHART_CURRENT_IMAGE_VERSION $CHART_NEW_IMAGE_VERSION) in
+  case $(vercmp $CHART_CURRENT_IMAGE_TAG $CHART_NEW_IMAGE_TAG) in
     "0" )
-      warn "Chart image \`${2}\` has not been updated."
+      warn "Chart image tag \`${CHART_NEW_IMAGE_TAG}\` has not been updated."
       return 1
       ;;
     "-1" )
-      info "Chart image \`${2%:*}:${CHART_CURRENT_IMAGE_VERSION}\` is newer than ${2}"
+      info "Current chart image tag \`${CHART_CURRENT_IMAGE_TAG}\` is newer than ${CHART_NEW_IMAGE_TAG}"
       return 1
       ;;
     "1" )
-      info "Updating chart image to '${2}'..."
-      sed -i 's|image: '"${2%:*}"':.*|image: '"${2}"'|' ${1}/values.yaml
-      git add ${1}/values.yaml
-      git commit -m "$CHART_NAME: update to \`${2}\`" >/dev/null
+      info "Updating chart image tag to '${CHART_NEW_IMAGE_TAG}'..."
+      sed -i "s|\(.*\)${CHART_CURRENT_IMAGE_TAG}|\1tag: ${CHART_NEW_IMAGE_TAG}|" ${CHART_PATH}/values.yaml
```

Comment:
In the past i have come across cases where I needed to properly escape shell variables in `sed` commands. You may need to change this to to avoid any possible issues

```
sed -i 's|\(.*\)'"${CHART_CURRENT_IMAGE_TAG}"'|\1tag: '"${CHART_NEW_IMAGE_TAG}'"|' ${CHART_PATH}/values.yaml
```